usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -155,14 +156,27 @@ def validate_request_data(request_data):
                 "array_max": maxsize,
             },
             {
-                "name": "last_id",
-                "key": "last_id",
+                "name": "last_record_unique_id",
+                "key": "last_record_unique_id",
+                "type": "integer",
+                "required": False,
+                "allow_nulls": True,
+            },
+            {
+                "name": "last_record_sort_value",
+                "key": "last_record_sort_value",
                 "type": "text",
                 "text_type": "search",
                 "required": False,
                 "allow_nulls": True,
             },
-            {"name": "last_value", "key": "last_value", "type": "float", "required": False, "allow_nulls": True},
+            {
+                "name": "last_record_tie_break_value",
+                "key": "last_record_tie_break_value",
+                "type": "float",
```

Comment:
If this is going to be `award_id`, should its type be `integer`? Either way, double check the types for all these fields.

I may have confused things when suggesting the names for these too. 

Effectively you're looking for a field too sort by, and a field that is unique, if the sort field is duplicated. The 2nd one (unique) is _also_ deemed the tie-breaker. Right? 

So it's possibly you only need fields like
1. `last_record_sort_value`
2. `last_record_unique_id` 

... and not mention anything about a tie-breaker field.

Reply:
I ended up adding a third sort field, since postgres has a default sort on the matviews (`total_obligation`), and in order to compare results I wanted to mimic that as close as possible. That's why this is a `float` not an `int`. I don't know if we want to keep that behavior, I don't know if that's something that we deliberately did or if it was done to try and speed up advanced search since the total_obligation sort is the default sort on the advanced search page.

Reply:
I ended up removing the third field, so disregard the previous comment